File: project_euler/10_sum_of_pri.rb
#!/usr/bin/ruby -w
# The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.

# Find the sum of all the primes below two million.

require 'prime'

i = 1
sum = 0
while i < 2000000
  if Prime.prime?(i)
    sum += i
  end
  i += 1
end
puts sum

File: project_euler/10_sum_of_pri.py
'''
# The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.

# Find the sum of all the primes below two million (2000000).
'''
import PrimeValidate

i = 2
limit = 2000000
sum = 0
while (i < limit):
  if (PrimeValidate.isPrime(i)): 
    sum += i
  i += 1
print sum

